### PR TITLE
Add Jamie

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by MarijeP** [comment]
Looks good to me. But then, I'm on grammar nazi.

File: _data/speakers.yml [hunk 1]
```diff
@@ -23,6 +23,48 @@ nonhighlighted: [  ]
...
+        name: Jamie Hewland
+        company: Praekelt Foundation
+        role: Site Reliability Engineer
+        twitter: jayhewland
+        github: JayH5
+        country: South Africa
+        bio: >
+          Jamie is a software developer at Praekelt Foundation. He believes
+          that good user experience is important throughout the full software
+          stack — from the front end down to the lowest-level system code.
```
Comment (vhata) [inline _data/speakers.yml]: This em-dash is unicode - make sure it generates properly plz.

**Comment by vhata** [comment]
:+1:

**Comment by adrianmoisey** [comment]
The schedule needs a talk description

**Comment by adrianmoisey** [comment]
> The schedule needs a talk description

No it doesn't.

**Pull request merged**